assets/wezterm-nautilus.py

```diff
@@ -44,17 +82,20 @@ def _open_terminal(self, path):
             # keep a separate CPU and memory account for Wezterm which in turn
             # ensures that oomd doesn't take nautilus down if a process in
             # wezterm consumes a lot of memory.
-            pid = int(child.get_identifier())
-            props = [("PIDs", GLib.Variant('au', [pid])),
-                ('CollectMode', GLib.Variant('s', 'inactive-or-failed'))]
-            name = f'app-nautilus-org.wezfurlong.wezterm-{pid}.scope'
-            args = GLib.Variant('(ssa(sv)a(sa(sv)))', (name, 'fail', props, []))
-            self._systemd.call_sync('StartTransientUnit', args,
-                    Gio.DBusCallFlags.NO_AUTO_START, 500, None)
+            self._systemd.move_to_dedicated_scope(int(child.get_identifier()))
+
+    def _open_paths_in_new_or_existing_terminal(self, paths):
+        if _wezterm_running():
+            for path in paths:
+                _open_in_existing_terminal(path)
+        else:
+            self._open_new_terminal(paths[0])
+            time.sleep(0.5)
```

Comment:
I'm not very happy about this sleep, but I haven't found a better way to wait for the fresh instance to start its internal mux server and be ready for wezterm cli connections.